README.md

```diff
@@ -1,3 +1,11 @@
-# Tezos tutorial applications
+# Tutorial application: Mint NFTs from a web application
```

Comment:
This replaces the readme at the root of tutorial-applications with a readme that is specific to the new application. Please restore this file and put the readme for the new application in /simple-nft-svelte/README.md.